charts/prefect-server/values.yaml

```diff
@@ -247,22 +247,21 @@ ingress:
   ##           port:
   ##             name: http
 
-
 # Postgresql configuration
 postgresql:
   enabled: true
   auth:
     # -- determines whether an admin user is created within postgres
     enablePostgresUser: false
-    # -- name for a custom database to create
+    # -- name for a custom database
     database: server
-    # -- name for a custom user to create
+    # -- name for a custom user
     username: prefect
     ## This is the password for `username` and will be set within the secret `{fullnameOverride}-postgresql` at the key `password`.
     ## This argument is only relevant when using the Postgres database included in the chart.
     ## For an external postgres connection, you must create and use `existingSecret` instead.
-    # -- password for the custom user to create. Ignored if `auth.existingSecret` with key `password` is provided
-    password: ""
+    # -- password for the custom user. Ignored if `auth.existingSecret` with key `password` is provided
+    password: "prefect-rocks"
```

Comment:
providing a default password so that chart install doesnt fail if the user doesn't make any changes